core/src/main/java/cz/xtf/core/bm/BinaryBuild.java

```diff
@@ -82,36 +162,69 @@ public void update(OpenShift openShift) {
 
 	@Override
 	public void delete(OpenShift openShift) {
-		openShift.deleteImageStream(is);
-		openShift.deleteBuildConfig(bc);
+		openShift.imageStreams().withName(is.getMetadata().getName()).delete();
+		openShift.buildConfigs().withName(bc.getMetadata().getName()).delete();
 	}
 
 	@Override
 	public boolean isPresent(OpenShift openShift) {
-		boolean isPresence = openShift.getImageStream(id) != null;
-		boolean bcPresence = openShift.getBuildConfig(id) != null;
+		boolean isPresence = openShift.imageStreams().withName(id).get() != null;
```

Comment:
What is the difference?

Reply:
I am trying to limit the API use to NamespacedOpenShiftClient whenever it makes sense.